src/components/Footer.js

```diff
@@ -8,6 +8,7 @@ const Footer = () => (
     <p className="text-hack-fg light-mode:text-hack-dark-title">
       Disclaimer: This site is fan made and not affiliated with{' '}
       <a
+      style={{color:"black"}}
```

Comment:
We use tailwind for styling, can you change this to a className attribute instead? Also, ideally want to re-use the colours declared in the tailwind config file. The colours are a bit of a mess and need sorting out, but in the mean time, you can probably use one of the colours there. See the surrounding p tag here for an example of some text colours and also the light-mode modifier.

Reply:
Sure I'll try to fix it